instance/models/openedx_instance.py

```diff
@@ -164,6 +163,12 @@ def get_active_appservers(self):
         """
         Returns a queryset containing the active appservers.
         """
+        if hasattr(self.ref, '_cached_active_appservers'):
+            # A database optimization like prefetch_related() hascomputed the active
+            # appservers for a large number of instances and cached that result on
+            # the InstanceReference of each one.
+            # (This is used to optimize the /api/v1/instances/ endpoint query for example)
+            return self.ref._cached_active_appservers
```

Comment:
Do we need to delete the cached appservers in any situatiion?  E.g. in the `OpenEdXAppServer.make_active()` method.